com.unity.render-pipelines.core/ShaderLibrary/API/Metal.hlsl

```diff
@@ -132,3 +132,8 @@
 #define GATHER_GREEN_TEXTURE2D(textureName, samplerName, coord2)          textureName.GatherGreen(samplerName, coord2)
 #define GATHER_BLUE_TEXTURE2D(textureName, samplerName, coord2)           textureName.GatherBlue(samplerName, coord2)
 #define GATHER_ALPHA_TEXTURE2D(textureName, samplerName, coord2)          textureName.GatherAlpha(samplerName, coord2)
+
+#if defined(SHADER_API_MOBILE)
+#define API_SUPPORTS_RENDER_PASS
```

Comment:
Other defined use naming like
#define PLATFORM_SUPPORTS_EXPLICIT_BINDING
#define PLATFORM_NEEDS_UNORM_UAV_SPECIFIER
#define PLATFORM_SUPPORTS_BUFFER_ATOMICS_IN_PIXEL_SHADER

Maybe PLATFORM_SUPPORTS_NATIVE_SUBPASSES?

Reply:
@hkr - metal wouldn't fit the description that well as it doesn't have subpasses. So I'll rename it to PLATFORM_SUPPORTS_NATIVE_RENDERPASS